src/pal/src/exception/seh.cpp

```diff
@@ -249,6 +266,9 @@ SEHProcessException(PAL_SEHException* exception)
                         PROCAbort();
                     }
                 }
+                
+                if(exceptionRecord->ExceptionFlags | EXCEPTION_ON_STACK)
+                    *exception = copyPAL_SEHException(exception);
```

Comment:
Could you please use the braces even for single line body to match the surrounding code style?